docs/src/threading.rst

```diff
@@ -90,7 +90,7 @@ Mutex locks
 Functions return 0 on success or an error code < 0 (unless the
 return type is void, of course).
 
-.. c:function:: int uv_mutex_init(uv_mutex_t* handle)
+.. c:function:: int uv_mutex_init(uv_mutex_t* handle, unsigned int flags)
```

Comment:
This would cause a lot of churn in the downstream ecosystem, with a bad cost/benefit ratio.

Can you make it a separate initializer function, e.g., `int uv_mutex_init_recursive(uv_mutex_t* handle);`?  That would also make it semver-minor instead of semver-major, i.e., it could land in the v1.x branch.